src/Search.php

```diff
@@ -791,8 +791,8 @@ public static function constructSQL(array &$data)
        //// 3 - WHERE
 
        // default string
-        $COMMONWHERE = self::addDefaultWhere($data['itemtype']);
-        $first       = empty($COMMONWHERE);
+        $COMMONWHERE = (Session::getCurrentInterface() != 'helpdesk') ? self::addDefaultWhere($data['itemtype']) : '';
```

Comment:
Does this mean we are removing the default clause for ALL itemtypes when on the helpdesk interface ?
If so I am afraid this might be too permissive.

Reply:
A little less permissive but worth a look  [545bd0f](https://github.com/glpi-project/glpi/pull/17471/commits/545bd0f8752fd8a084e3670fa78ac3a047e09f1a)